### DISCUSSION TITLE
[Somewhat of a feature suggestion, but possibly more a suggestion as-such] Consider integrating diffend.io functionality

### DISCUSSION THREAD

**Discussion by rubyFeedback** [rubygems/rubygems.org#3434] [Ideas]
I have been using https://my.diffend.io/gems/ quite a lot and I like it.

We can easily compare changed functionality between gems (at the least
the changed code), sometimes find whether we made a mistake and what
not. I use it quite a bit for small incremental changes, to see what changed
and may still need change. It's an additional "focus lens" for me on my
code/gems.

So, why not integrate this into rubygems.org directly?

With that I mean, rather than:

    https://my.diffend.io/gems/name_of_the_gem/1.2.3/.1.24

We would have this on a per-gem variant at rubygems.org, such as:

    https://rubygems.org/name_of_the_gem/1.2.3/.1.24

That's the main part of the proposal. Ideally we could also add some
more fine-tuning to the output and what not, to allow people to 
change it, focus on what they prefer and so forth.

I don't know how the people at diffend.io feel about it, but I wanted
to maket his proposal anyway, and perhaps they can chime in what
they think about it.

(Both websites could offer the same functionality, but I would rather
see it integrated into rubygems.org as-is and then be more amenable
to any change; with it being external we may lose it one day, and 
we may not as easily optimize or adjust it. Or perhaps I didn't know
how to customize it yet, that may also be the case.)

**Comment by simi** [comment]
there's [RFC](https://github.com/rubygems/rfcs/pull/44) "in progress", feel free to comment that one
